receiver/mongodbatlasreceiver/internal/metadata/metric_name_mapping.go

```diff
@@ -0,0 +1,1001 @@
+// Copyright  OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package metadata
+
+import (
+	"fmt"
+	"strings"
+	"time"
+
+	"go.mongodb.org/atlas/mongodbatlas"
+	"go.opentelemetry.io/collector/model/pdata"
+)
+
+type metricMappingData struct {
+	metricName string
+	attributes map[string]pdata.AttributeValue
+}
+
+var metricNameMapping = map[string]metricMappingData{
+	// MongoDB CPU usage. For hosts with more than one CPU core, these values can exceed 100%.
+
+	// sfx: process.cpu.user
+	"PROCESS_CPU_USER": {"process.cpu.utilization", map[string]pdata.AttributeValue{
+		"state":       pdata.NewAttributeValueString("user"),
```

Comment:
I'm not familiar with Atlas, but are there distinguishing values in the measurement that can be used?  Having a ~callback for the measurement and metric similar to the initializer function for settings attributes may be helpful if so.

Reply:
Unfortunately much of the attribute definitions here come from the name of the measurement, and the names aren't quite consistent enough to be parsable for the information. The measurement data structure does have the unit name, but there's nothing I can really use to fold multiple measurements into a single metric.